### PR TITLE
Disable New Relic and RabbitMQ by default

### PR DESCRIPTION
**Pull request by rocioar**
Disables New relic and RabbitMQ depends on: https://github.com/open-craft/configuration/pull/43/

### CODE REVIEW

**Comment by bradenmacdonald** [comment]
Are we still using New Relic on some instances?

**Comment by clemente** [comment]
@bradenmacdonald It seems this only disables the [newrelic role](https://github.com/edx/configuration/blob/master/playbooks/roles/newrelic/tasks/main.yml), which is for `nrsysmond` which was deprecated.
`COMMON_ENABLE_NEWRELIC` is used in many places, but all seem to relate to the `newrelic` role.

**Comment by bradenmacdonald** [comment]
I see, thanks.

File: instance/models/mixins/openedx_config.py [hunk 1]
```diff
@@ -246,6 +246,9 @@ def _get_configuration_variables(self):
             "DISCOVERY_NGINX_PORT": 80,
             "DISCOVERY_SSL_NGINX_PORT": 443,
 
+            # RabbitMQ disabled locally
+            "SANBOX_ENABLE_RABBITMQ": False,
```
Comment (clemente) [inline instance/models/mixins/openedx_config.py]: SANBOX → SANDBOX

**Comment by UmanShahzad** [comment]
@rocioar @clemente Shouldn't this be merged soon?

**Comment by pomegranited** [comment]
@rocioar CC @UmanShahzad  Looks like the [`SANBOX_ENABLE_RABBITMQ` typo](https://github.com/open-craft/opencraft/pull/317/files/5f7c4a39073f56f9abb4be0dec998b551702d83f#diff-b3cc82fdb60b23de6fff2d4252452568) that @clemente  noted didn't get fixed before this was merged 😦

**Pull request merged**